### PR TITLE
Update tcld reference to v0.1.3

### PR DESCRIPTION
**Pull request by djmagee**
## Preview

https://deploy-preview-1207--mystifying-fermi-1bc096.netlify.app/cloud/tcld/

## What does this PR do?

Update docs to v0.1.3

- In `tcld namespace`
  - Move subcommands of `update` up one level
  - Remove `update` subcommand
  - Add `add`, `list`, and `remove` subcommands
- Add `--auto_confirm` modifier to `tcld`

## Notes to reviewers

How-to for cert rollover scheduled for later PR

### CODE REVIEW

File: docs/cloud/tcld/index.md [hunk 1]
```diff
@@ -23,3 +23,12 @@ The Temporal Cloud CLI (tcld) is a command-line tool that you can use to interac
...
 - [`tcld request`](/cloud/tcld/request)
 - [`tcld version`](/cloud/tcld/version)
+
+## Global modifier
+
+### `--auto_confirm`
+
+Automatically confirm all prompts.
+
+You can specify the value for this modifier by setting the AUTO_CONFIRM environment variable.
```
[the remark this answers is not in the document]
Comment (djmagee) [reply docs/cloud/tcld/index.md:33]: At the moment, AUTO_CONFIRM is the only environment variable I see for tcld. If tcld begins using others, I'll likely create a separate page similar to tctl's.

**Review by mattkim — APPROVED** [review]

File: docs/cloud/tcld/namespace/search-attributes/rename.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+```bash
+tcld namespace search-attributes rename --resource-version <etag> --existing-name <value> --new-name <value>
+```
+
+### `--existing-name`
+
+_Required modifier_
+
+Specify the name of an existing Search Attribute.
```
[the remark this answers is not in the document]
Comment (djmagee) [reply docs/cloud/tcld/namespace/search-attributes/rename.md:57]: Both the tcld reference and the tctl reference consistently use imperative verbs.

File: docs/cloud/tcld/namespace/accepted-client-ca/add.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+`tcld namespace accepted-client-ca add --ca-certificate <value>`
+
+Alias: `a`
+
+The following modifiers control the behavior of the command.
+
+### `--namespace`
+
+Specify a Namespace hosted on Temporal Cloud. If not specified, the value of the environment variable $TEMPORAL_CLOUD_NAMESPACE is used.
```
[the remark this answers is not in the document]
Comment (djmagee) [reply docs/cloud/tcld/namespace/accepted-client-ca/add.md:21]: For anywhere other than a pure reference page, I agree. To keep the text compact, I used the more elliptical wording throughout.

File: docs/cloud/tcld/namespace/search-attributes/rename.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+id: rename
+title: tcld namespace search-attributes rename
+sidebar_label: rename
+description: How to rename an existing custom Search Attribute in Temporal Cloud using tcld.
+tags:
+  - reference
+  - tcld
+---
+
+The `tcld namespace search-attributes rename` command renames an custom [Search Attributes](/concepts/what-is-a-search-attribute) in Temporal Cloud.
```
[the remark this answers is not in the document]
Comment (djmagee) [reply docs/cloud/tcld/namespace/search-attributes/rename.md:11]: _D'oh!_

Comment (djmagee) [inline docs/cloud/tcld/namespace/search-attributes/rename.md:11]:
```suggestion
The `tcld namespace search-attributes rename` command renames a custom [Search Attribute](/concepts/what-is-a-search-attribute) in Temporal Cloud.
```

**Pull request merged**